packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -485,3 +485,8 @@ export function isInstanceOfISnapshot(
 ): obj is ISnapshot {
 	return obj !== undefined && "snapshotFormatV" in obj && obj.snapshotFormatV === 1;
 }
+
+/**
+ * Default GroupId to ask for snapshot fetch when no groupId is specified.
+ */
+export const defaultGroupIdForSnapshot = "default";
```

Comment:
I'd guess it should be "", i.e. empty, but not sure - could you please follow up with SPO?

Reply:
/latest?ump=1&groupId=default (ungrouped only, “partial”) 
…/latest?ump=1&groupId=g1 (requesting single group) 
…/latest?ump=1&groupId=g1Ab-cDe (requesting single group) 
…/latest?ump=1&groupId=g1A,g2B,g3c (requesting group g1A, g2B, g3c) 

As per the design.